### PR TITLE
Improved sleeper UI

### PR DESCRIPTION
**Pull request by DamianX**
Because I was tired of scrolling and resetting and scrolling and resetting and scrolling and resetting when the sleeper was upgraded.
image

:cl:
 * tweak: Updated the medical sleeper UI.

### CODE REVIEW

**Comment by Ephx1** [comment]
Does the "Refresh meter readings each second" now actually do what it's supposed to?

**Comment by DamianX** [comment]
no

**Comment by Ephx1** [comment]
Well I guess you can't have everything. Still a :+1: though, glad to see this change happen.

**Comment by EdXbtkOEwGw** [comment]
This pallette is awful

**Comment by DamianX** [comment]
changes welcome

**Comment by DamianX** [comment]
Or how's this? 
image
The only colors are white and orangeish

**Comment by EdXbtkOEwGw** [comment]
Much better

**Comment by PJB3005** [comment]
> Not NanoUI.

**Comment by DamianX** [comment]
/vg/UI when

**Comment by gbasood** [comment]
I actually liked red for dead but I didn't like the shade of red specifically

File: code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -39,62 +39,60 @@
...
+		dat += "<b>Occupant statistics:</b><BR>"
+		if (occupant)
+			var/occupant_status = "???"
+			switch(occupant.stat)
+				if(CONSCIOUS)
+					occupant_status = "conscious"
+				if(UNCONSCIOUS)
+					occupant_status = "<span class='average'>unconscious</span>"
+				if(DEAD)
+					occupant_status = "<span class='average'>*dead*</span>"
```
Comment (Intigracy) [inline code/game/machinery/Sleeper.dm:66]: `<span class='warning'>` would probably look like what you're looking for

Comment (DamianX) [reply code/game/machinery/Sleeper.dm:66]: Isn't that a goonchat thing?

Comment (PJB3005) [reply code/game/machinery/Sleeper.dm:66]: It is.

**Pull request merged**